modules/light-clients/08-wasm/types/config.go

```diff
@@ -6,10 +6,11 @@ const (
 	// contractMemoryLimit is the memory limit of each contract execution (in MiB)
 	// constant value so all nodes run with the same limit.
 	ContractMemoryLimit = 32
+	// The wasm vm cache size is set to 0 to reduce unnecessary memory usage. See: https://github.com/CosmWasm/cosmwasm/pull/1925
+	MemoryCacheSize = 0
```

Comment:
nit: godocs for exported symbols should start with the name of the symbol 

(maybe if you decide to fix this you could fix above `ContractMemoryLimit`'s godoc is lower camel case when it should be pascal case

Reply:
ah, yeah. Gladly fixing both.